src/status_im/navigation/screens.cljs

```diff
@@ -635,6 +636,12 @@
                 :insets                 {:top? true}}
     :component wallet-swap-select-asset-to-pay/view}
 
+   {:name      :screen/wallet.swap-from
```

Comment:
I would suggest a more descriptive name, for example `:screen/wallet.swap-select-account` (and update the namespace accordingly)

Reply:
done